### ISSUE TITLE
Incorrect links in Operator Installation manual

### ISSUE THREAD

**Issue by NerRox** [keycloak/keycloak#11576]
### Describe the bug

Hi! I am trying to install KeyCloak operator to vanilla k8s, but links on this pages are not correct
 https://www.keycloak.org/operator/installation

I'm receiving 404 error instead of k8s manifests.

### Version

18.0.0

**Comment by NerRox** [comment]
The links are

kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/18.0.0/kubernetes/keycloaks.k8s.keycloak.org-v1.yml
kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/18.0.0/kubernetes/keycloakrealmimports.k8s.keycloak.org-v1.yml



kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/18.0.0/kubernetes/kubernetes.yml

**Comment by matheushent** [comment]
[This](https://github.com/keycloak/keycloak-operator/tree/main/deploy) is the link for the correct operator installation

**Comment by ruckc** [comment]
@matheushent - those links point to the old operator... the release notes say the new operator is a rewrite from scratch and versioned at 20.x, though I can't seem to find the new operator code.

**Comment by andreaTP** [comment]
This is correct, the PR for the release of the resources has not been merged yet, please refer to the resources in this PR:
https://github.com/keycloak/keycloak-k8s-resources/pull/1

cc. @stianst

**Comment by vmuzikar** [comment]
> @matheushent - those links point to the old operator... the release notes say the new operator is a rewrite from scratch and versioned at 20.x, though I can't seem to find the new operator code.

Thank you for your interest in the new operator! The code can be found [here](https://github.com/keycloak/keycloak/tree/main/operator).

**Comment by vmuzikar** [comment]
With https://github.com/keycloak/keycloak-k8s-resources/pull/1 meged, the links should be working now. Closing this.
